src/System.Linq/src/System/Linq/Enumerable.cs

```diff
@@ -3931,12 +3959,12 @@ public TElement LastOrDefault()
 
         public TElement[] ToArray()
         {
-            return new TElement[0]; // consider replacing with Array.Empty<TElement>()
+            return Array.Empty<TElement>();
         }
 
         public List<TElement> ToList()
         {
-            return new List<TElement>(0);
+            return new List<TElement>();
```

Comment:
It's only in there because back when .NET was closed-source I picked up the habit of using `(0)` when creating a list that was expected to remain empty, because it seems like a reasonable choice from the outside, though examining the source shows `()` is slightly more efficient. It's reasonable to tidy it up while optimising the memory use of the similar `ToArray()`.